README.md

```diff
@@ -626,6 +626,25 @@ An up-to-date list of the convenience operations can be found [here](https://doc
 | GET    | /v1/contextTypes                                                                     | R |
 | GET    | /v1/contextTypes{typename}                                                           | R ]
 
+#### NGSIv2 
+| Method | Path                                                                                    | Action |
+| ------ |:--------------------------------------------------------------------------------------- | ---:|
+| GET    | /v2                                                                                     | R |
+| GET    | /v2/entities                                                                            | R |
+| GET    | /v2\/entities/{entityId}                                                                | R |
+| POST   | /v2\/entities                                                                           | C |
+| PATCH  | /v2\/entities/{entityId}                                                                | U |
+| POST   | /v2\/entities/{entityId}                                                                | U |
```

Comment:
Options are dealt with in the Orion plugin. The solution you mention was
the first approach, replaced to allow multiple options and quey parameters.
El 27/10/2015 19:44, "Fermín Galán Márquez" <notifications@github.com>
escribió:

> In README.md
> <https://github.com/telefonicaid/fiware-pep-steelskin/pull/261#discussion_r43167928>
> :
>
> > @@ -626,6 +626,25 @@ An up-to-date list of the convenience operations can be found [here](https://doc
> >  | GET    | /v1/contextTypes                                                                     | R |
> >  | GET    | /v1/contextTypes{typename}                                                           | R ]
> >
> > +#### NGSIv2
> > +| Method | Path                                                                                    | Action |
> > +| ------ |:--------------------------------------------------------------------------------------- | ---:|
> > +| GET    | /v2                                                                                     | R |
> > +| GET    | /v2/entities                                                                            | R |
> > +| GET    | /v2\/entities/{entityId}                                                                | R |
> > +| POST   | /v2\/entities                                                                           | C |
> > +| PATCH  | /v2\/entities/{entityId}                                                                | U |
> > +| POST   | /v2\/entities/{entityId}                                                                | U |
>
> What about replacing
>
> | POST   | /v2\/entities/{entityId}                                                                | U |
>
> with
>
> | POST   | /v2\/entities/{entityId}                                                                | U |
> | POST   | /v2\/entities/{entityId}?options=append                                                 | C |
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/telefonicaid/fiware-pep-steelskin/pull/261/files#r43167928>
> .
>